src/markdown.cpp

```diff
@@ -2790,8 +2796,13 @@ size_t Markdown::Private::writeBlockQuote(std::string_view data)
     if (curLevel != 0 || !isGitHubAlert)
     {
       QCString txt = data.substr(indent,end-indent);
-      if (txt.stripWhiteSpace().isEmpty() && !startCmd.isEmpty()) out += "<br><br>\n";
+      if (txt.stripWhiteSpace().isEmpty() && !startCmd.isEmpty())
+      {
+        if (!isGitHubFirst) out += "<br>";
```

Comment:
This starts to look more and more fiddly, can we not solve this in CSS by using some margins and maybe a different class for the GitHub alerts (if really needed)?

Reply:
It indeed looks like a bit fiddly but there are some considerations to do it this way:
- the problem does not only show in HTML but also in e.g. LaTeX so just a css approach is not sufficient
- the "empty line" is not always present and thus the first `<br>` can be at a different place and would be harder to detect with css (when it is anyway possible at that moment).

I didn't see a better solution than adding this `isGitHubFirst` variable.